tests/README.md

```diff
@@ -1,57 +1,37 @@
-@page testing Testing Guide
+# Testing Guide
 
-[TOC]
+[Integration testing](https://en.wikipedia.org/wiki/Integration_testing) is an essential part of software development. It is performed when integrating code changes into the main development branch to verify that the code works as expected. The following sections describe how to run and add integration tests used to the svFSIplus program.
 
-[Integration testing](https://en.wikipedia.org/wiki/Integration_testing) is an essential part of software development. It is performed when integrating code changes into the main development branch to verify that the code works as expected. Below is a quick guide on how to run and add integration tests for `svFSI`.
+Running a test case requires 
+- Build svFSIplus
+- Install Git LFS used to download test data
```

Comment:
Do you think it makes sense to invert the order, putting "install gti lfs" as first item? 
I don't know exactly how git lfs works but I noticed that it would be better to install it before cloning or at least before building to avoid any problems.

Reply:
The svFSIplus build process and `git lfs` are not connected, just need to run `git lfs` before running the tests.